app/views/tag/_subscribe_button.html.erb

```diff
@@ -1,13 +1,11 @@
 <!-- This button is used to display tags in a popover and subscribe to multiple tags using a single button -->
 <% if current_user %>
-  <a class="btn btn-default btn-xs sub" rel="popover" data-container="body" data-placement="bottom" data-template='<div class="popover" role="tooltip"><div class="arrow"></div><div class="popover-body"><%= render partial: 'tag/tags_popover', locals: {tags: tags, label_name: 'label-info'} %></div></div>' data-html="true" data-content="sample" > <i class="fa fa-plus"></i> Subscribe</a>
+  <a class="btn btn-primary btn-xs sub" rel="popover" data-container="body" data-placement="bottom" data-template='<div class="popover" role="tooltip"><div class="arrow"></div><div class="popover-body"><%= render partial: 'tag/tags_popover', locals: {tags: tags} %></div></div>' data-html="true" data-content="sample" > <i class="fa fa-plus fa-white"></i> Subscribe>> </a>
 <% else %>
-  <a class="btn btn-default btn-xs sub" data-toggle="modal" data-target="#loginModal" role="button" aria-pressed="true"> <i class="fa fa-plus"></i> Subscribe</a>
+  <a class="btn btn-primary btn-xs sub" data-toggle="modal" data-target="#loginModal" role="button" aria-pressed="true"> <i class="fa fa-plus fa-white"></i> Subscribe >></a>
```

Comment:
use class requireLogin so that the pop over will open at soon as the login is executed for not logged in user.

Reply:
Why is the login modal flickering on adding this class? Or is it just mine?

Reply:
We want the button to be clicked automatically after clicking on the Subscribe+ if user is not logged in.
requireLogin is made by Kevin+Jonathan for such kind of work. 
This will enable user not to double click on the button.

Reply:
Is this problem just at my localhost?

![Uploading flicker.gif…]()

Reply:
Ohh actually it was flickering like this on my local host thats why I asked:
![flicker](https://user-images.githubusercontent.com/40794215/51208950-09ce8000-1935-11e9-9a80-c4f2f7cf19be.gif)

Reply:
So remove this. No need to add erroneous things.
Thanks for the effort

Reply:
Okay, I haven't added it anyways.